admin_guide/image_policy.adoc

```diff
@@ -39,6 +39,20 @@ in the integrated container registry
 You can configure registries allowed for import in *_master-config.yaml_*
 under `imagePolicyConfig:allowedRegistriesForImport` section as demonstrated in
 the following example. If the setting is not present, all images are allowed.
+The xref:../install_config/install/advanced_install.adoc[advanced installation]
+does not generate the setting, therefor it whitelists all registries. However,
+the OpenShift master API spawned without any configuration file will use the
+recommended list of registries.
+
+The full recommended list of registries can be listed by writing out the
+configuration file:
+
+====
+dir="$(mktemp -d)"
+openshift start master --write-config="${dir}" && \
+  sed -n '/^\s*allowedRegistriesForImport/,/^\s*[a-z]/p' "${dir}/master-config.yaml" && \
+  rm -rf "${dir}"
+====
```

Comment:
Just two tiny things, otherwise LGTM. 

s/therefor/therefore 
The ==== [note box] here should be ---- [code block]. 

I'll open a follow-up PR to correct these. :)